modules/cnv-connecting-resource-bridge-network.adoc

```diff
@@ -0,0 +1,85 @@
+// Module included in the following assemblies:
+//
+// * cnv/cnv-attaching-vm-multiple-networks.adoc
+
+[id="connecting-resource-bridge-network-{context}"]
+= Connecting a resource to a bridge-based network
+
+As a network administrator, you can configure a NetworkAttachmentDefinition
+of type `cnv-bridge` to provide Layer-2 networking to Pods and virtual machines.
+
+.Prerequisites
+
+* {ProductName} 2.0 or newer
+* A Linux bridge must be configured and attached to the correct
+Network Interface Card on every node.
+* If you use VLANs, `vlan_filtering` must be enabled on the bridge.
+* The NIC must be tagged to all relevant VLANs.
+** For example: `bridge vlan add dev bond0 vid 1-4095 master`
+
+.Procedure
+
+. Create a new file for the NetworkAttachmentDefinition in any local directory.
+The file must have the following contents, modified to match your
+configuration:
++
+[source,yaml]
+----
+apiVersion: "k8s.cni.cncf.io/v1"
+kind: NetworkAttachmentDefinition
+metadata:
+  name: a-bridge-network
+spec:
+  config: '{
+    "cniVersion": "0.3.0",
+    "name": "a-bridge-network", <1>
+    "type": "cnv-bridge", <2>
+    "bridge": "br0", <3>
+    "isGateway": true,
+    "ipam": {}
+}'
+----
+<1> The `name` value is part of the annotation you will use in the next step.
+<2> The actual name of the Container Network Interface (CNI) plug-in that provides
+the network for this NetworkAttachmentDefinition. Do not change this field unless
+you want to use a different CNI.
+<3> You must substitute the actual name of the bridge, if it is not `br0`.
+
+. Edit the configuration file of a resource that you want to connect to the
+bridge network:
++
+[source,yaml]
+----
+apiVersion: v1
+kind: Pod
```

Comment:
Don't we prefer to use a _VirtualMachine_ yaml for our examples, instead of _Pod_? This is a CNV document.

Reply:
Good question. We do usually focus on VMs and VMIs in our examples, but I think it's good to show a variety of options. @dankenigsberg what do you think?

Reply:
I think that VM would be better, but I would not loose sleep over it. I can live with Pod in this single occurance.